client/index.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import App from './src/App.jsx'
+
+ReactDOM.render(<App />, document.getElementById('app'));
```

Comment:
thought we agreed not to use App?

Reply:
from the lecture video, it looks as though the what you call it doesn't matter. Fred had 4 different src calling 4 different api end points called App. Can change if you want

Reply:
not a big deal at all just curious! its all approved :+)